etl/scripts-ccao-data-raw-us-east-1/spatial/spatial-kriging_surfaces.R

```diff
@@ -595,36 +630,39 @@ create_year_kriging <- function(data, year_num, raster_file){
   clean_data <- unique(clean_data)
 
 
-  v = variogram(noise~ 1, clean_data, cutoff = 200000, width = 500)
-  v.m = fit.variogram(v, vgm(1, "Gau", 5000, 1))
-  k = krige(noise ~ 1, clean_data, raster_file, v.m)
-  #pdf(str_c("output/krig_demo/", year_num, ".pdf"), width = 11, height = 8.5)
+  v <- variogram(noise ~ 1, clean_data, cutoff = 200000, width = 500)
+  v.m <- fit.variogram(v, vgm(1, "Gau", 5000, 1))
+  k <- krige(noise ~ 1, clean_data, raster_file, v.m)
+  # nolint start
+  # pdf(str_c("output/krig_demo/", year_num, ".pdf"), width = 11, height = 8.5)
 
-  #plot_surface(k)
-
-  #dev.off()
+  # plot_surface(k)
 
+  # dev.off()
+  # # nolint end
```

Comment:
**[Nitpick, non-blocking]** Rather than turn off linting for this block, maybe we just delete the unused code that's commented out?